docs/resources/server/advancements.md

```diff
@@ -231,7 +231,7 @@ builder.rewards(
     // Alternatively, use addExperience() to add to an existing builder.
     AdvancementRewards.Builder.experience(100)
     // Alternatively, use loot() to create a new builder.
-    .addLootTable(ResourceLocation.fromNamespaceAndPath("minecraft", "chests/igloo"))
+    .addLootTable(ResourceKey.create(Registries.LOOT_TABLE, ResourceLocation.fromNamespaceAndPath("minecraft", "blocks/dirt")))
```

Comment:
Would you mind changing this on the 1.20.6 docs as well? I missed this there.

Reply:
I didn't see a part in the 1.20.6 docs where the `.addLootTable()` method is mentionned

Reply:
My apologies, I thought that the server resources rework was merged before 1.21. This can be resolved.